packages/material-ui-lab/src/TreeItem/TreeItem.test.js

```diff
@@ -255,395 +289,655 @@ describe('<TreeItem />', () => {
+
+          fireEvent.click(getByText('one'));
+          expect(getByTestId('one')).to.have.attribute('aria-expanded', 'true');
+          // move focus to node two
+          fireEvent.click(getByText('two'));
+          fireEvent.click(getByText('two'));
+          expect(getByTestId('two')).to.have.attribute('aria-expanded', 'false');
+          fireEvent.keyDown(document.activeElement, { key: 'ArrowLeft' });
+          expect(getByTestId('one')).to.have.focus;
+          expect(getByTestId('one')).to.have.attribute('aria-expanded', 'true');
+        });
+
+        it('should do nothing if focus is on a root node that is closed', () => {
+          const { getByTestId } = render(
+            <TreeView>
+              <TreeItem nodeId="one" label="one" data-testid="one">
+                <TreeItem nodeId="two" label="two" />
+              </TreeItem>
+            </TreeView>,
+          );
 
-        expect(getByTestId('one')).to.have.attribute('aria-expanded', 'true');
-        getByTestId('one').focus();
-        fireEvent.keyDown(document.activeElement, { key: 'ArrowRight' });
-        expect(getByTestId('two')).to.have.focus;
+          getByTestId('one').focus();
+          expect(getByTestId('one')).to.have.attribute('aria-expanded', 'false');
+          fireEvent.keyDown(document.activeElement, { key: 'ArrowLeft' });
+          expect(getByTestId('one')).to.have.focus;
+        });
+
+        it('should do nothing if focus is on a root node that is an end node', () => {
+          const { getByTestId } = render(
+            <TreeView>
+              <TreeItem nodeId="one" label="one" data-testid="one" />
+            </TreeView>,
+          );
+
+          getByTestId('one').focus();
+          fireEvent.keyDown(document.activeElement, { key: 'ArrowLeft' });
+          expect(getByTestId('one')).to.have.focus;
+        });
       });
 
-      it('should do nothing if focus is on an end node', () => {
-        const { getByTestId, getByText } = render(
-          <TreeView defaultExpanded={['one']}>
-            <TreeItem nodeId="one" label="one" data-testid="one">
+      describe('down arrow interaction', () => {
+        it('moves focus to a non-nested sibling node', () => {
+          const { getByTestId } = render(
+            <TreeView>
+              <TreeItem nodeId="one" label="one" data-testid="one" />
               <TreeItem nodeId="two" label="two" data-testid="two" />
-            </TreeItem>
-          </TreeView>,
-        );
-
-        fireEvent.click(getByText('two'));
-        expect(getByTestId('two')).to.have.focus;
-        fireEvent.keyDown(document.activeElement, { key: 'ArrowRight' });
-        expect(getByTestId('two')).to.have.focus;
+            </TreeView>,
+          );
+
+          getByTestId('one').focus();
+          fireEvent.keyDown(document.activeElement, { key: 'ArrowDown' });
+          expect(getByTestId('two')).to.have.focus;
+        });
+
+        it('moves focus to a nested node', () => {
```

Comment:
I would prefer the WAI-ARIA terminology and use 
```suggestion
        it('moves focus to a child node', () => {
```